Plugin.cpp

```diff
@@ -24,6 +24,18 @@ BINARYNINJAPLUGIN bool CorePluginInit()
     Workflow::registerActivities();
     Commands::registerCommands();
 
+    BinaryNinja::Ref<BinaryNinja::Settings> settings = BinaryNinja::Settings::Instance();
+    settings->RegisterGroup("objc", "Objective-C");
+
+    settings->RegisterSetting("objc.cleanupARCCode",
```

Comment:
The setting group key should be "workflows.objectiveC" to better (but not identically) match the workflow identifier and also group it under the workflows group in settings.